packages/runtime/runtime-utils/src/test/summarizerNode.spec.ts

```diff
@@ -330,6 +330,7 @@ describe("Runtime", () => {
                 it("Should refresh from tree when no proposal handle provided", async () => {
                     createRoot();
                     const result = await rootNode.refreshLatestSummary(
+                        undefined,
```

Comment:
You can use the sequenceNumber of the "attributes" blob defined in the test:
const blobs = {
                protocolAttributes: { sequenceNumber: 123 },
            } as const;